src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -69,11 +71,21 @@ class CodyAgentService(project: Project) : Disposable {
       agent.client.onTextDocumentShow = Function { params ->
         CodyEditorUtil.showDocument(
             project,
-            params.uri,
+            URI.create(params.uri).withScheme("file"),
```

Comment:
so the scheme can still be `untitled` here but we are overwriting it anyway, right?

Reply:
Yes, `untitled` is VSCode thing which leaks to use a little bit.
Those are really a files, just not created on a disk yet.